pkg/csi/cinder/nodeserver.go

```diff
@@ -545,17 +547,33 @@ func getAvailabilityZoneMetadataService(m openstack.IMetadata) (string, error) {
 	return zone, nil
 }
 
-func getNodeID(mount mount.IMount, metadata openstack.IMetadata) (string, error) {
+func getNodeID(mount mount.IMount, iMetadata openstack.IMetadata, order string) (string, error) {
 	// First try to get instance id from mount provider
-	nodeID, err := getNodeIDMountProvider(mount)
-	if err == nil {
-		return nodeID, nil
+	if len(order) == 0 {
```

Comment:
It would be good if we could set the default order somewhere where config is initialized , so that it could be reused.